### ISSUE TITLE
brise: missing *.prism.bin

### ISSUE THREAD

**Issue by ixearth** [void-linux/void-packages#10888]
### System

* xuname:  
  *Void 4.19.34_1 x86_64-musl GenuineIntel uptodate rFFFF*
* package:  
  *brise-0.38.20180515_1*

### Expected behavior
RIME schema should include *.prism.bin but it only contains *.yaml and a essay.txt
check brise on arch
https://www.archlinux.org/packages/community/x86_64/brise/
also some separated packages on debian
https://packages.debian.org/stretch/amd64/librime-data-bopomofo/filelist
maybe it's dictionary files of RIME schema, and probably necessary for a RIME schema functional.

### Actual behavior
anyway, RIME does not works for me on void (both ibus-rime and fcitx-rime does not work for me). maybe it's not the only reason.
https://github.com/rime/brise/
it says brise is deprecated and they recommend to use plum. IDK about plum but seems non of any distro package plum yet.

and currently brise is a dependencies of ibus-rime, but fcitx-rime does not have brise as dependencies.

but I think at least this missing "*.prism.bin" problem should be fixed, or the brise package could be useless.

### Steps to reproduce the behavior

```
$ xbps-query -Rf brise
/usr/share/rime-data/array30.dict.yaml
/usr/share/rime-data/array30.schema.yaml
/usr/share/rime-data/bopomofo.schema.yaml
/usr/share/rime-data/bopomofo_express.schema.yaml
/usr/share/rime-data/bopomofo_tw.schema.yaml
/usr/share/rime-data/cangjie5.dict.yaml
/usr/share/rime-data/cangjie5.schema.yaml
/usr/share/rime-data/cangjie5_express.schema.yaml
/usr/share/rime-data/combo_pinyin.schema.yaml
/usr/share/rime-data/combo_pinyin_kbcon.schema.yaml
/usr/share/rime-data/combo_pinyin_left.schema.yaml
/usr/share/rime-data/default.yaml
/usr/share/rime-data/double_pinyin.schema.yaml
/usr/share/rime-data/double_pinyin_abc.schema.yaml
/usr/share/rime-data/double_pinyin_flypy.schema.yaml
/usr/share/rime-data/double_pinyin_mspy.schema.yaml
/usr/share/rime-data/double_pinyin_pyjj.schema.yaml
/usr/share/rime-data/emoji.dict.yaml
/usr/share/rime-data/emoji.schema.yaml
/usr/share/rime-data/essay.txt
/usr/share/rime-data/hkcantonese.schema.yaml
/usr/share/rime-data/ipa_xsampa.dict.yaml
/usr/share/rime-data/ipa_xsampa.schema.yaml
/usr/share/rime-data/jyutping.dict.yaml
/usr/share/rime-data/jyutping.schema.yaml
/usr/share/rime-data/luna_pinyin.dict.yaml
/usr/share/rime-data/luna_pinyin.schema.yaml
/usr/share/rime-data/luna_pinyin_fluency.schema.yaml
/usr/share/rime-data/luna_pinyin_simp.schema.yaml
/usr/share/rime-data/luna_pinyin_tw.schema.yaml
/usr/share/rime-data/luna_quanpin.schema.yaml
/usr/share/rime-data/pinyin_simp.dict.yaml
/usr/share/rime-data/pinyin_simp.schema.yaml
/usr/share/rime-data/quick5.dict.yaml
/usr/share/rime-data/quick5.schema.yaml
/usr/share/rime-data/sampheng.schema.yaml
/usr/share/rime-data/scj6.dict.yaml
/usr/share/rime-data/scj6.schema.yaml
/usr/share/rime-data/soutzoe.dict.yaml
/usr/share/rime-data/soutzoe.schema.yaml
/usr/share/rime-data/stenotype.schema.yaml
/usr/share/rime-data/stroke.dict.yaml
/usr/share/rime-data/stroke.schema.yaml
/usr/share/rime-data/symbols.yaml
/usr/share/rime-data/terra_pinyin.dict.yaml
/usr/share/rime-data/terra_pinyin.schema.yaml
/usr/share/rime-data/wubi86.dict.yaml
/usr/share/rime-data/wubi86.schema.yaml
/usr/share/rime-data/wubi_pinyin.schema.yaml
/usr/share/rime-data/wubi_trad.schema.yaml
/usr/share/rime-data/wugniu.schema.yaml
/usr/share/rime-data/wugniu_lopha.dict.yaml
/usr/share/rime-data/wugniu_lopha.schema.yaml
/usr/share/rime-data/yale.schema.yaml
/usr/share/rime-data/zyenpheng.dict.yaml
/usr/share/rime-data/zyenpheng.schema.yaml
```

**Comment by maxice8** [comment]
We believe this is fixed on brise-0.38.20180515_2

**Comment by ixearth** [comment]
yeah. rime still make fcitx crash tho, I guess that's related to librime or fcitx-rime.
